phase1-coordinator/src/objects/round.rs

```diff
@@ -783,13 +794,38 @@ impl Round {
         }
     }
 
+    /// Remove a contributor from the round.
+    pub(crate) fn remove_contributor_unsafe(
```

Comment:
I hope we will remove "_unsafe" and "try_" from the function names one day. In case of `remove_something_unsafe` it's misleading because it's a safe function. And `try_do_something() -> Result<...>` doesn't need to have "try_" because it returns `Result` and it's clear that it can fail. And here the name is justified by the similarity to other `remove_*_unsafe` functions, which is ok for now.

Reply:
I agree, I was just following convention in this case. I think perhaps `try_` doesn't quite convey the intended meaning. The function may not fail with a result, but my guess is the original authors were trying to say that calling this function might be dangerous in the wrong contexts (without being "unsafe" according to the strict Rust definition).